config/database.yml

```diff
@@ -1,25 +1,14 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
   pool: 5
-  timeout: 5000
+  username: <%= `whoami` %>
+  password:
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: haxxor-news_development
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
-
-production:
-  <<: *default
-  database: db/production.sqlite3
+  database: haxxor-news_test
```

Comment:
Cool I'll add it to .gitignore. I'm not sure if I completely understanding how my code is checking that information (or simply not explicitly stating not to), but perhaps we can discuss that in person/hangout.